<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>610</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>정선동강</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Motion Palette</string>
  </property>
  <widget class="QFrame" name="Main_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1001</width>
     <height>571</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>641</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>정선동강</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Motion Palette&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_prev">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>75</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>이전</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_next">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>230</y>
      <width>75</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>다음</string>
    </property>
   </widget>
   <widget class="QLabel" name="GV_main">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>640</width>
      <height>340</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Master Pices</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GV_sub_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Master Pices</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GV_sub_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Master Pices</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GV_sub_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Master Pices</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GV_sub_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Master Pices</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="GV_sub_main">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Master Pices</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_com">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>400</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>선택하기</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
